spec/fixtures/pages/form-with-data.html

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
please put this fixture in `spec-main`

Reply:
Wasn't sure of the convention, since all the other fixtures used in `chromium-spec.ts` (via  `fixturesPath`) point to `spec/`. Should I change this one or all of them in `chromium-spec.ts`? I guess I don't see the value in separating them if there's a chance they'll be used in both.